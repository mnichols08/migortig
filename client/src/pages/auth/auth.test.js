import React from 'react';
import { shallow } from 'enzyme';
import AuthPage from './index';

it('should render AuthPage component', () => {
  expect(shallow(<AuthPage />)).toMatchSnapshot();
});
