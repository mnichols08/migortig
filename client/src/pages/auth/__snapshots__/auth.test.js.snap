// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AuthPage component 1`] = `ShallowWrapper {}`;
